content/en/seachat/seachat-manual/04-channels/07-seachat-voicebot.md

```diff
@@ -0,0 +1,117 @@
+<img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/channels/voicebot/choose-inbound-calls.png" alt="">
+</a>
+
+*Setting up **Inbound Calls** Channel*
+</center>
+
+### Toll-Free Number
+
+In the **Inbound Calls** channel, we need to first purchase a toll-free phone number. Choose the country where you want to purchase the toll-free number and phone number. Once you have chosen the desirable configuration,  click on the **Confirm Purchase** button.
+
+<br/>
+<center>
+  <a href="/images/product-updates/seachat/en/channels/voicebot/buy-a-number.png" style="height: 200px; width: 100%; height: 100%;display: flex; justify-content: center; align-items: center; overflow: hidden;" target="_blank">
+<img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/channels/voicebot/buy-a-number.png" alt="">
+</a>
+
+*Purchase a toll-free number*
+</center>
+
+Once purchased, you can see the toll-free number of your choice. You can always release the number and purchase a new one if you are not satisfied with the number.
+
+<br/>
+<center>
+  <a href="/images/product-updates/seachat/en/channels/voicebot/toll-free-number.png" style="height: 200px; width: 100%; height: 100%;display: flex; justify-content: center; align-items: center; overflow: hidden;" target="_blank">
+<img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/channels/voicebot/toll-free-number.png" alt="">
+</a>
+
+*Toll-free number details*
+</center>
+
+### Agent Configuration
+
+After purchasing the toll-free number, we need to configure the agent to handle the inbound calls. In the **Configure Voice Agent** section, you can set up the voice agent to handle the inbound calls. 
+
+You are welcome to enable the live agent feature for your voice agent just like any other SeaChat agent. Please make sure you have a real-human agent that will monitor and respond to the user when the live agent feature is enabled.
+
+<br/>
+<center>
+  <a href="/images/product-updates/seachat/en/channels/voicebot/configure-agent.png" style="height: 200px; width: 100%; height: 100%;display: flex; justify-content: center; align-items: center; overflow: hidden;" target="_blank">
+<img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/channels/voicebot/configure-agent.png" alt="">
+</a>
+
+*Voice agent configuration*
+</center>
+
+As you can see in the image above, there are some parameters that need to be set up to fully configure the voice agent. If you are not sure about the configuration, you can start from choosing the language and SeaChat will automatically set up the rest of the configuration for you. As you can see in the image below, I have chosen the English language with SeaVoice-2 model for my voice agent. Now click on **Save Configuration** to consolidate your settings.
```

Comment:
please use SeaVoice not SeaVoice-2. also put in a few words to encourage people to use SeaVoice and SeaVoice-2 is just experimental and unstable.

and also update the screenshot.